Description: allow manually generating channels/h323/Makefile.ast
 This patch is a work-around to that bug. It allows running
 'make h323-mak' from debian/rules to generate
 channels/h323/Makefile.ast before running the main 'make' command.
Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Bug: https://issues.asterisk.org/view.php?id=17162
Last-Update: 2016-03-19

--- a/Makefile
+++ b/Makefile
@@ -965,6 +965,9 @@
 asterisk.txt:
 	$(MAKE) -C doc/tex asterisk.txt
 
+h323-mak:
+	$(MAKE) -C channels h323/Makefile.ast
+
 .PHONY: menuselect
 .PHONY: main
 .PHONY: sounds
